delivery_carrier_label_postlogistics/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# © 2013-2015 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
Lines above not useful, there is no real code in this file, and it's in ascii, the defaut coding